internal/nbd/nbd_test.go

```diff
@@ -89,6 +90,12 @@ while [ $# -gt 0 ]; do
 			;;
 		-c)
 			if [ -e %[4]s ]; then
+				i=0
+				while [ ! -e %[5]s ] && [ $i -lt 100 ]; do
+					i=$((i+1))
+					sleep 0.001
+				done
+				rm -f %[5]s
```

Comment:
Consider adding some documentation before `qemunbdScriptTpl` to explain what this script is doing and how it is used by the tests.